.gitattributes

```diff
@@ -1,2 +1,3 @@
 .gitattributes export-ignore
 .gitignore export-ignore
+CHANGES.current merge=union
```

Comment:
The CHANGES.current merge conflicts are a real nuisance so we do need a better solution, but this ought to go into a separate pull request. I don't quite know if it will work, the git documentation https://git-scm.com/docs/gitattributes says it will merge in a random order, but we want the dates to be done in correct date/time order.

Reply:
Sure thing, I will cut this apart and research if we can implement a custom merge strategy that would preserve ordering of the timestamps in the entries.